internal/usecase/interactor/auth.go

```diff
@@ -74,8 +76,31 @@ func NewAuthStorage(cfg *StorageConfig, request repo.AuthRequest, getUserBySubje
 			PostalCode:         cfg.DN.PostalCode,
 		}
 	}
+	c, err := config.Load(ctx)
+	if err != nil {
+		panic("Could not load auth config")
```

Comment:
```suggestion
		log.Panicf("Could not load auth config: %s", err)
```

Note that I commented that NewAuthStorage should return errors instead of panicking